whelk-core/src/main/resources/ext/marcframe.json

```diff
@@ -5940,14 +5940,15 @@
       "include": ["identifier"],
       "match": [
         {
-          "when": "i1=7 & $2 =~ ansi|ANSI|doi|DOI|hdl|HDL|isan|ISAN|iso|ISO|istc|ISTC|iswc|ISWC|uri|URI|urn|URN",
+          "when": "i1=7 & $2 =~ ansi|ANSI|danacode|doi|DOI|hdl|HDL|isan|ISAN|iso|ISO|istc|ISTC|iswc|ISWC|uri|URI|urn|URN",
```

Comment:
Check for uppercase `DANACODE` too?

Reply:
The code is lowercase and I checked _some_ examples, all where lowercase.

Reply:
There is _one_ exception: https://libris.kb.se/3gc7x0vh1vnn5v99 :slightly_smiling_face: